src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -56,30 +65,198 @@ public function getLockOwner(): ?string
 
     /**
      * Get the entry funding link
-     *
-     * @psalm-return null|object{url: string, title: string}
      */
     public function getFunding(): ?stdClass
+    {
+        return $this->getPodcastIndexFunding();
+    }
+
+    /**
+     * Get the entry funding link
+     */
+    public function getPodcastIndexFunding(): ?stdClass
     {
         if (array_key_exists('funding', $this->data)) {
-            return $this->data['funding'];
+            /** @var stdClass $object */
+            $object = $this->data['funding'];
+            return $object;
         }
 
         $funding = null;
 
         $nodeList = $this->xpath->query($this->getXpathPrefix() . '/podcast:funding');
 
         if ($nodeList->length > 0) {
+            /** @var DOMElement $item */
+            $item           = $nodeList->item(0);
```

Comment:
```suggestion
            $item = $nodeList->item(0);
            assert($item instanceof DOMElement);
```

Assertions will break in tests, but get dropped in production

Reply:
Oh, I was not aware that using assertions is a practiced way to ensure type safety. If you prefer that, I will change it for all the get methods.